package-lock.json

```diff
@@ -39331,7 +39331,7 @@
 				"cross-env": "^7.0.3",
 				"csurf": "1.11.0",
 				"date-fns": "2.28.0",
-				"date-fns-tz": "*",
+				"date-fns-tz": "^2.0.0",
```

Comment:
Isn't this going to be over-written with npm install?

Reply:
npm install will read a package-lock.json file and use the version defined in it. If adding a new package or something, it'll update the package-lock.